### ISSUE TITLE
Top-level await error on build

### ISSUE THREAD

**Issue by ericfennis** [TaTo30/vue-pdf#132]
Hi Thank you for making `vue-pdf`.

I get an error when trying to build the app:
```
[vite:esbuild-transpile] Transform failed with 1 error:
assets/Attachment-!~{0b_}~.js:26353:53: ERROR: Top-level await is not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14" + 3 overrides)

Top-level await is not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14" + 3 overrides)
26351|  /******/ // This entry module used 'module' so it can't be inlined
26352|  /******/ var __webpack_exports__ = __webpack_require__([22](https://github.com/nedap/arsis/actions/runs/10196742478/job/28208100554?pr=609#step:5:23)8);
26353|  /******/ __webpack_exports__ = globalThis.pdfjsLib = await (globalThis.pdfjsLibPromise = __webpack_exports__);
   |                                                       ^
26354|  /******/ __webpack_exports__.AbortException;
26355|  /******/ __webpack_exports__.AnnotationEditorLayer;
```

Seems going wrong in the pre-compiled `pdfjs` package included in NPM package. Is there a reason why this is included in the package?

**Additional context**
 - vue-pdf: "4.3.0"
 - vue: "3.4.7"
 
 **Temporary fix**
Use [vite-plugin-top-level-await](https://github.com/Menci/vite-plugin-top-level-await) to transpile the top-level await.

**Comment by QvQQ** [comment]
me too, using Nuxt.js 3

**Comment by TaTo30** [comment]
Duplicate #113

**Comment by ericfennis** [comment]
@TaTo30 I've tried the fix from #113, but get a different error: 
```
ERROR: Top-level await is not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14" + 3 overrides)

Top-level await is not available in the configured target environment ("chrome87", "edge88", "es2020", "firefox78", "safari14" + 3 overrides)
26353|  /******/ // This entry module used 'module' so it can't be inlined
26354|  /******/ var __webpack_exports__ = __webpack_require__(228);
26355|  /******/ __webpack_exports__ = globalThis.pdfjsLib = await (globalThis.pdfjsLibPromise = __webpack_exports__);
   |                                                       ^
26356|  /******/ __webpack_exports__.AbortException;
26357|  /******/ __webpack_exports__.AnnotationEditorLayer;
```
